src/net.cpp

```diff
@@ -2108,6 +2108,7 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
             {
                 bool notify = false;
                 if (!pnode->ReceiveMsgBytes({pchBuf, (size_t)nBytes}, notify)) {
+                    LogPrint(BCLog::NET, "receiving message bytes failed for peer=%d\n", pnode->GetId());
```

Comment:
This one didn't get "disconnecting" added?

Reply:
No, because `CloseSocketDisconnect` already uses that term. I updated the PR description (yesterday) to explain the reasoning here.

Reply:
(This is currently being resolved by adding a call to `DisconnectMsg()` unless I've missed something).

https://github.com/bitcoin/bitcoin/blob/06443b8f28bcec4315cec262eb03343dee5465a6/src/net.cpp#L2146-L2153